packages/serverpod/lib/src/server/serverpod.dart

```diff
@@ -203,6 +207,70 @@ class Serverpod {
     _logManager = LogManager(settings, serverId: serverId);
   }
 
+  /// Initializes the servers internal shutdown task managers and registers
+  /// shutdown tasks.
+  ///
+  /// This method is called during server startup and sets up the servers
+  /// internal task managers with all the necessary tasks to properly shutdown
+  /// the server.
+  void _initializeShutdownTaskManagers() {
+    _requestReceivingShutdownTasks.addTask(
+      Task(
+        'Server',
+        server.shutdown,
+      ),
+    );
+
+    _requestReceivingShutdownTasks.addTask(
+      Task(
+        'Web Server',
+        () async => _webServer?.stop(),
+      ),
+    );
+
+    _requestReceivingShutdownTasks.addTask(
+      Task(
+        'Service Server',
+        () async => _insightsServer?.shutdown(),
+      ),
+    );
+
+    _requestReceivingShutdownTasks.addTask(
+      Task(
+        'Future Call Manager',
+        () async => _futureCallManager?.stop(),
+      ),
+    );
+
+    _internalServicesShutdownTasks.addTask(
+      Task(
+        'Test Auditor',
+        () async => _shutdownTestAuditor(),
+      ),
+    );
+
+    _internalServicesShutdownTasks.addTask(
+      Task(
+        'Internal Session',
+        _internalSession.close,
+      ),
+    );
+
+    _internalServicesShutdownTasks.addTask(
+      Task(
+        'Redis Controller',
+        () async => redisController?.stop,
```

Comment:
```suggestion
        () async => redisController?.stop(),
```

Reply:
Also, does it make sense to register a task that does not do anything?

I think the existence of the `redisController` is fixed by this point, so we could just not add the task.

Not that it costs much, but might be confusing if the inspector/logger shows it when it does not seem used.